src/resolvers/FlyerResolver.ts

```diff
@@ -238,6 +243,32 @@ class FlyerResolver {
       startDate,
       title,
     );
+    let editedResponse = '';
+    if ((endDate != null && location != null) || (location != null && startDate != null)) {
+      editedResponse = editedResponse.concat('changed its date and location');
+    } else if (endDate != null && startDate != null) {
+      editedResponse = editedResponse.concat('changed its date');
+    } else if (endDate != null) {
+      const date = new Date(endDate);
+      editedResponse = editedResponse.concat(
+        'changed its end date to '.concat(date.toDateString()),
+      );
+    } else if (location != null) {
+      if (editedResponse !== '') {
+        editedResponse = editedResponse.concat(' and ');
+      }
+      editedResponse = editedResponse.concat('changed its location to '.concat(location));
+    } else if (startDate != null) {
+      const date = new Date(startDate);
+      if (editedResponse !== '') {
+        editedResponse = editedResponse.concat(' and ');
+      }
+      editedResponse = editedResponse.concat(
+        'changed its start date to '.concat(date.toDateString()),
+      );
+    }
+    NotificationRepo.notifyFlyersForOrganizations(flyer.id, editedResponse, 'e');
+    return flyer;
```

Comment:
Summary for this section:
1. For this entire section, can you space things out and add comments describing what each `if` branch does?
2. I don't know if you need a space before `changed` for these but something to watch out for.
3. Instead of checking if the variable is not equal to null, you can just use the variable. For example, if (endDate != null) is equivalent to if (endDate).
4. Instead of using `.concat`, I would use template strings. For example, `changed its start date to ${date.toDateString())`